core-tests/src/test/java/org/mule/runtime/core/util/AttributeEvaluatorTestCase.java

```diff
@@ -34,144 +40,156 @@
 
   public static final DataType INTEGER = DataType.fromType(Integer.class);
   private ExtendedExpressionManager mockExpressionManager = mock(ExtendedExpressionManager.class);
-  private Event mockMuleEvent = mock(Event.class);
+  private Event event = mock(Event.class);
 
   @Test
   public void plainTextValue() {
     AttributeEvaluator attributeEvaluator = new AttributeEvaluator("attributeEvaluator");
     when(mockExpressionManager.isExpression("attributeEvaluator")).thenReturn(false);
     attributeEvaluator.initialize(mockExpressionManager);
-    assertThat(attributeEvaluator.isParseExpression(), is(false));
-    assertThat(attributeEvaluator.isExpression(), is(false));
+
```

Comment:
Remove this TestCase due that is a duplicated one from `MelAttributeEvaluatorTestCase`